scaling_performance/cluster_limits.adoc

```diff
@@ -111,6 +113,12 @@ Kubernetes scheduler makes during pod placement. Learn what measures you can
 take to xref:../admin_guide/overcommit.adoc#disabling-swap-memory[avoid memory swapping].
 ====
 
+[IMPORTANT]
+====
+Some of the limits are streched only in a single dimension, so the they might
+vary when lot of objects are running on the cluster.
```

Comment:
when a lot of

Reply:
"so the they" -> "so they"
"objects running on the cluster" - is there a better way to phrase it? objects don't really run on the cluster.